cmd/skopeo/delete.go

```diff
@@ -3,44 +3,52 @@ package main
 import (
 	"errors"
 	"fmt"
-	"io"
 	"strings"
 
 	"github.com/containers/image/v5/transports"
 	"github.com/containers/image/v5/transports/alltransports"
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 )
 
 type deleteOptions struct {
 	global *globalOptions
 	image  *imageOptions
 }
 
-func deleteCmd(global *globalOptions) cli.Command {
-	sharedFlags, sharedOpts := sharedImageFlags()
-	imageFlags, imageOpts := imageFlags(global, sharedOpts, "", "")
```

Comment:
(Overall, this would be notably easier to review if it didn’t move unchanged code around, or if there were a separate commit with code changes, and then a separate commit with moves and no changes.)